core/src/main/java/discord4j/core/spec/TextChannelEditSpec.java

```diff
@@ -110,7 +110,7 @@ public TextChannelEditSpec setParentId(@Nullable Snowflake parentId) {
 
     /**
      * Sets the amount of seconds a user has to wait before sending another message to the modified
-     * {@link TextChannel}, from 0 to 120. Does not affect bots or users with {@link Permission#MANAGE_MESSAGES} or
+     * {@link TextChannel}, from 0 to 21600 seconds. Does not affect bots or users with {@link Permission#MANAGE_MESSAGES} or
```

Comment:
If we can't find an official source mentioning this limit we should remove it. Otherwise we should include the link along with this fix.

Reply:
This is in https://discordapp.com/developers/docs/resources/channel#modify-channel in field "rate_limit_per_user", maybe mention see the docs for limits in the url.